.github/workflows/main.yml

```diff
@@ -108,6 +108,61 @@ jobs:
           if-no-files-found: error
 
 
+  java-cluster:
+    name: Experimental LSP job on JDK 8
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '11' ]
+      fail-fast: false
+    steps:
+        
+      - name: Set up JDK ${{ matrix.java }} 
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ matrix.java }} 
+          distribution: 'zulu'
+
+      - name: Setup Xvfb
+        if: ${{ matrix.java != '20-ea' }}  # see #4299
+        run: |
+          echo "DISPLAY=:99.0" >> $GITHUB_ENV
+          Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+
+      - name: Checkout ${{ github.ref }} ( ${{ github.sha }} )
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          submodules: false
+
+      - name: Caching dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.hgexternalcache
+          key: ${{ runner.os }}-${{ hashFiles('*/external/binaries-list', '*/*/external/binaries-list') }}
+          restore-keys: ${{ runner.os }}-
+
+      - name: Build Java Cluster
+        run: |
+          ant $OPTS -quiet -Dcluster.config=java -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false clean
+          ant $OPTS -quiet -Dcluster.config=java -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false build
+        
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: 'zulu'
+
+      - name: java/java.lsp.server
+        run: ant $OPTS -Dcluster.config=java -Djavac.compilerargs=-nowarn -Dbuild.compiler.deprecation=false -f java/java.lsp.server test
```

Comment:
not sure if it matters for this test. But maybe inline the $OPTS here. Since I forgot a '-' where it is declared which causes yaml to add a newline at the end of the list.
```
  OPTS: >-
    -Dmetabuild.jsonurl=https://raw.githubusercontent.com/apache/netbeans-jenkins-lib/master/meta/netbeansrelease.json
    -Dtest-unit-sys-prop.ignore.random.failures=true
```
would be correct (note the `-` after `>`)

it is fixed in the original PR, forgot to add it here.